### ISSUE TITLE
SA1503 fails to report `lock` statement without braces

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1184]
The following statement should result in SA1503, but currently does not produce any warning.

```csharp
lock (this)
    Console.WriteLine();
```
